htp/htp_transaction.c

```diff
@@ -1453,7 +1453,7 @@ htp_status_t htp_tx_state_response_headers(htp_tx_t *tx) {
                                 "Compression bomb: multiple encoding with lzma");
                         break;
                     }
-                } else if (bstr_util_cmp_mem(tok, tok_len, "inflate", 7) == 0) {
+                } else if (bstr_util_cmp_mem(tok, tok_len, "inflate", 7) == 0 || bstr_util_cmp_mem(tok, tok_len, "none", 4) == 0) {
```

Comment:
should these comparisons be stricter? I seems like "infateme" and "noneother" would match here as well due to the limited length check

Reply:
Do you want to ?

Reply:
I bet something can be crafted, but not easily